packages/tofu/src/ses-whitelist.js

```diff
@@ -1,3 +1,5 @@
+// @ts-nocheck
+
```

Comment:
(could) document why the bailout

Reply:
I'd be even happier if we, say, moved this into a `vendor/` dir and kept it up-to-date with `ses`.

Reply:
Not in this PR, but yes. Let's even look into exporting it from `ses`

Reply:
created #814